### DISCUSSION TITLE
Get back to Fedora EPEL repository

### DISCUSSION THREAD

**Discussion by tjyang** [pnp4nagios/pnp4nagios#44] [Ideas]

**Comment by tjyang** [comment]
https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2015-6615



![Image](https://github.com/pnp4nagios/pnp4nagios/assets/378638/214b750c-7c19-4b2d-b366-d10e6e3b4e8d)

**Reply by celane** [reply]
Now that automation is working, can certainly build EPEL8 rpms. What does it take to get them into the Fedora EPEL repository?

**Reply by tjyang** [reply]
- The original packager(ondrejj) for EL7 is still active doing EL packaging. See [https://bodhi.fedoraproject.org/users/ondrejj](https://bodhi.fedoraproject.org/users/ondrejj). 
- Quick search found this URL https://docs.fedoraproject.org/en-US/epel/epel-package-request/ , Need to figure out how to file an official request to activate pnp4ngios for EL8+.

**Reply by tiahino** [reply]
I am basing my internal package on EPEL7 spec. Trust me - the changes made to the installation process (by 0.6.26-4l at least) do not make building the rpm in sane environment easier.
